microsoft-edge/webview2/concepts/process-related-events.md

```diff
@@ -0,0 +1,636 @@
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [COREWEBVIEW2_PROCESS_FAILED_REASON enum](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/webview2-idl#corewebview2_process_failed_reason)
+
+---
+
+
+<!-- ---------- -->
+###### Exit code, process description, and frames information
+
+In addition to _failure kind_ and _failure reason_, the `ProcessFailed` event provides more detailed information about the reported failure, including:
+* Exit code
+* Process description (utility only)
+* Frames information (renderer only)
+
+Your application can leverage this information for diagnostics and other scenarios. For more information about when these details are provided and how your application can use them, see:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2ProcessFailedEventArgs Class](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2processfailedeventargs)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2ProcessFailedEventArgs Class](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2processfailedeventargs)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2ProcessFailedEventArgs2 interface](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2processfailedeventargs2)
+
+---
+
+
+<!-- ---------- -->
+###### Gathering details
+
+Your application can use and collect information from the above events to identify the most frequent issues for your application.
+
+Some process failures might raise the `ProcessFailed` event across different WebView2 controls in your application; see [Handle unexpected exits for various types of processes](#handle-unexpected-exits-for-various-types-of-processes).  You must decide how often to gather details and how to handle duplicates for these cases.
+
+Additionally, most process crashes will generate dumps in the [user data folder](./user-data-folder.md), under the directory returned by `FailureReportFolderPath`. You can use these dumps to understand crashes and provide additional information when contacting the WebView2 team.
+
+##### [.NET/C#](#tab/dotnetcsharp)
+
+* [CoreWebView2Environment.FailureReportFolderPath Property](https://learn.microsoft.com/dotnet/api/microsoft.web.webview2.core.corewebview2environment.failurereportfolderpath)
+
+##### [WinRT/C#](#tab/winrtcsharp)
+
+* [CoreWebView2Environment.FailureReportFolderPath Property](https://learn.microsoft.com/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#failurereportfolderpath)
+
+##### [Win32/C++](#tab/win32cpp)
+
+* [ICoreWebView2Environment11::get_FailureReportFolderPath](https://learn.microsoft.com/microsoft-edge/webview2/reference/win32/icorewebview2environment11#get_failurereportfolderpath)
+
+---
+
+
+<!-- ------------------------------ -->
+#### Handle unexpected exits for various types of processes
+
+
+<!-- ---------- -->
+###### Types of processes
```

Comment:
Can we just make this say "Before reading further, you should understand the WebView Process Model" linking to the doc? This is a lot of content to summarize another page, I don't like having two places to maintain the same topic.

Reply:
so far, I started the parent section "Handle unexpected exits for various types of processes" by saying:
To read this section, you should understand the [Process model for WebView2 apps](process-model.md).

Kept the dup info, where it's designed as a nav-linked summary list.

Reply:
I confirmed the above status.  Discuss in meeting whether to keep the below detail.

Reply:
Added a See Also link near top & bottom of article:
[Process model for WebView2 apps](./process-model.md)
@lflores-ms - you'll be able to move this link if you want.  after i push this commit